config.js

```diff
@@ -480,6 +480,8 @@ config.EVENT_LEVEL = 5;
 config.LOG_TO_STDERR_ENABLED = true;
 config.LOG_TO_SYSLOG_ENABLED = false;
 
+config.LOG_COLOR_ENABLED = true;
```

Comment:
I did not see any use of this config, Do we need it?

Reply:
done